consensus/state_sync/module.go

```diff
@@ -1,18 +1,25 @@
 package state_sync
 
 import (
+	typesCons "github.com/pokt-network/pocket/consensus/types"
 	"github.com/pokt-network/pocket/logger"
 	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
 	"github.com/pokt-network/pocket/shared/modules"
 )
 
 const (
-	stateSyncModuleName = "stateSyncModule"
+	stateSyncModuleName       = "stateSyncModule"
+	committedBlocsChannelSize = 100
 )
 
 type StateSyncModule interface {
 	modules.Module
 	StateSyncServerModule
+
+	Set(aggregatedMetaData *typesCons.StateSyncMetadataResponse)
+	CommittedBlock(uint64)
```

Comment:
`CommittedBlock` sounds like something that happend rather than a function anyhoe would call

Reply:
removed, using event bus.